ssdutil/main.py

```diff
@@ -16,10 +17,46 @@
 
 DEFAULT_DEVICE="/dev/sda"
 SYSLOG_IDENTIFIER = "ssdutil"
+DISK_TYPE_SSD = "0"
+DISK_INVALID = "-1"
 
 # Global logger instance
 log = logger.Logger(SYSLOG_IDENTIFIER)
 
+def get_default_disk():
+    """Check default disk"""
+    default_device = DEFAULT_DEVICE
+    host_mnt = '/host'
+    cmd = "lsblk -l -n |grep {}".format(host_mnt)
+    proc = subprocess.Popen(cmd, shell=True, text=True, stdout=subprocess.PIPE)
+    out = proc.stdout.readline()
+    if host_mnt in out:
```

Comment:
Is this check necessary? `grep` should take care of this. Perhaps check if the result is not empty (or grep return code)?